src/core/ext/client_channel/client_channel.h

```diff
@@ -38,6 +38,9 @@
 #include "src/core/ext/client_channel/resolver.h"
 #include "src/core/lib/channel/channel_stack.h"
 
+// Channel arg key for server URI string.
+#define GRPC_ARG_SERVER_URI "grpc.server_uri"
```

Comment:
Not sure what you're asking here.  It's the URI of the server, the same value used when creating the channel on the client.